ffi/jpeg.lua

```diff
@@ -0,0 +1,93 @@
+--[[--
+Module for JPEG decoding/encoding.
+
+This is a LuaJIT FFI wrapper for turbojpeg.
+
+@module ffi.jpeg
+]]
+
+local BB = require("ffi/blitbuffer")
+local ffi = require("ffi")
+
+local _ = require("ffi/turbojpeg_h")
+local turbojpeg = ffi.load("turbojpeg")
```

Comment:
Well, apparently, the hoop-jumping was necessary, c.f., the testsuite:

```
[----------] Running tests from ./spec/base/unit/pic_spec.lua
./ffi/jpeg.lua:13: libturbojpeg.so: cannot open shared object file: No such file or directory
```

(ping @Frenzie/@poire-z).

Reply:
It'd work in combination with front where cpath is set and ffi.load is replaced with a wrapper. But indeed this is base.

Reply:
Aaah, that makes sense, thanks!

Reply:
What does that mean? Can I leave `local turbojpeg = ffi.load("turbojpeg")`?

Reply:
No you can't, as it at the very least breaks the testsuite ;).